image_services/rtc_services/rtc_service_cleanup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+#pull any remote changes
+
+git pull origin main
```

Comment:
I'm not sure what the working directory will be when run via `cron` (probably `/` or `/home/arcgis/`?), so I think we'll need a `cd /home/arcgis/gis-services/` here